config/locales/searchworks.en.yml

```diff
@@ -82,13 +88,14 @@ en:
       form:
         q: ''
       zero_results:
-        modify_your_search: "Modify your search"
+        modify_your_search: "Modify your search in %{search_type}"
         page_heading: "No results"
-        limit_fields: "Remove limit(s): "
-        search_fields: "Search all fields: "
-        your_search: "Your search: "
+        limit_fields: "Remove limit(s)"
+        search_fields: "Search all fields"
+        your_search: "Your current search"
         return_to_advanced_search: "Return to Advanced search page"
         limit: "Try using fewer terms to start, then limit your results."
+        search_context: "Try your search in..."
```

Comment:
Should this be the elipsis character (`…`) rather than 3 periods?

Reply:
✅ thanks! I didn't know an ellipsis character existed.